spec/pom.xml

```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>org.eclipse.microprofile.reactive-streams-operators</groupId>
         <artifactId>microprofile-reactive-streams-operators-parent</artifactId>
-        <version>2.1-SNAPSHOT</version>
+        <version>3.0-SNAPSHOT</version>
```

Comment:
I believe the properties and build can be removed.

Reply:
nice thx